src/fw/system/bootbits.c

```diff
@@ -102,32 +102,37 @@ uint32_t boot_version_read(void) {
 
 #elif MICRO_FAMILY_SF32LB52
 
-// TODO(SF32LB52): implement
-
-static uint32_t _bootbits;
-
 void boot_bit_init(void) {
-  _bootbits = BOOT_BIT_INITIALIZED;
+  rtc_init();
```

Comment:
Since there no pebble bootloader for sf32lb52 now, `boot_bit_init` currently is inactive.

Reply:
boot_bit_init() is actually never called by PebbleOS, am I right? If so, we should maybe call it _somewhere_, at least on SF32LB52 until we have a bootloader?

Reply:
I'd add in main (after rtc init?) something like

```
// FIXME(SF32LB52): remove once we have bootloader
#ifdef MICRO_FAMILY_SF32LB52
boot_bit_init();
#endif
```

Reply:
Both is ok, will add to main latter.